Add errored result counts and filter parameter details to Policy Checks API docs (#172)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/policy-checks.mdx

|----------------------------------- | ----------------------------------------------------------------------- |
| `page[number]`                     | **Optional.** If omitted, the endpoint returns the first page.          |
| `page[size]`                       | **Optional.** If omitted, the endpoint returns 20 agent pools per page. |

> | `filter[n][status]`                | **Optional.** If omitted, the endpoint returns all policies regardless of status. Must be one of: passed, failed, errored. |
> | `filter[n][enforcementLevel]`      | **Optional.** Only used if paired with a non-errored status filter. Must be one of: advisory, mandatory. |

Reviewer: Should these be code font instead? I just want to set them apart from the typical text a bit.

```suggestion
| `filter[n][status]`                | **Optional.** If omitted, the endpoint returns all policies regardless of status. Must be either "passed", "failed", or "errored". |
| `filter[n][enforcementLevel]`      | **Optional.** Only used if paired with a non-errored status filter. Must be either "advisory" or "mandatory." |
```

Author: I think in quotes as suggested is good, thank you.

> -> **Note**: The `filter[n]` is a zero indexed filter and can combine filtering combinations of statuses and enforcement levels. Policy outcomes with an errored status do not have an enforcement level.

Reviewer: ```suggestion
-> **Note**: You can use `filter[n]` to combine combinations of statuses and enforcement levels. Policy outcomes with an errored status do not have an enforcement level.
```
